projection/block_raw_event/block_raw_event.go

```diff
@@ -0,0 +1,138 @@
+package blockevent
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/crypto-com/chain-indexing/appinterface/projection/rdbprojectionbase"
+	"github.com/crypto-com/chain-indexing/appinterface/rdb"
+	event_entity "github.com/crypto-com/chain-indexing/entity/event"
+	"github.com/crypto-com/chain-indexing/entity/projection"
+	applogger "github.com/crypto-com/chain-indexing/external/logger"
+	"github.com/crypto-com/chain-indexing/external/utctime"
+	"github.com/crypto-com/chain-indexing/infrastructure/pg/migrationhelper"
+	"github.com/crypto-com/chain-indexing/projection/block_raw_event/view"
+	event_usecase "github.com/crypto-com/chain-indexing/usecase/event"
+)
+
+var _ projection.Projection = &BlockRawEvent{}
+
+type BlockRawEvent struct {
+	*rdbprojectionbase.Base
+
+	rdbConn rdb.Conn
+	logger  applogger.Logger
+
+	migrationHelper migrationhelper.MigrationHelper
+}
+
+func NewBlockRawEvent(
+	logger applogger.Logger,
+	rdbConn rdb.Conn,
+	migrationHelper migrationhelper.MigrationHelper,
+) *BlockRawEvent {
+	return &BlockRawEvent{
+		rdbprojectionbase.NewRDbBase(
+			rdbConn.ToHandle(),
+			"BlockRawEvent",
+		),
+
+		rdbConn,
+		logger,
+
+		migrationHelper,
+	}
+}
+
+func (_ *BlockRawEvent) GetEventsToListen() []string {
+	return []string{
+		event_usecase.RAW_BLOCK_EVENT_CREATED,
```

Comment:
`BlockRawEventCreated` vs `RAW_BLOCK_EVENT_CREATED`

choose one naming for alignment